dcore/state.py

```diff
@@ -72,11 +76,19 @@ def dump(self):
 
         funcs = self.xn.split()
         to_dump = []
+
         for name, f in zip(self.fieldlist, funcs):
             if name in self.output.dumplist:
                 to_dump.append(f)
             f.rename(name=name)
 
+        for field, meanfield in zip(funcs, self.output.meanfields):
```

Comment:
I think we're going to need a more flexible way of adding diagnostic fields soon but it this ok for now? I've just realised I could use the same setup to check steady state solutions but putting the initial field into the meanfields list (in which case maybe I should rename meanfields...)